overseer/src/lib.rs

```diff
@@ -121,6 +121,35 @@ enum OverseerMessage<M: Debug, I> {
 	},
 }
 
+enum Event {
+	BlockImport,
+	BlockFinalized,
+	Stop,
+}
+
+/// A handler used to communicate with the `Overseer`.
+#[derive(Clone)]
+pub struct OverseerHandler {
+	events_tx: mpsc::Sender<Event>,
+}
+
+impl OverseerHandler {
+	/// Inform the `Overseer` that that some block was imported.
+	pub async fn block_imported(&mut self) -> SubsystemResult<()> {
+		self.events_tx.send(Event::BlockImport).await.map_err(|_| SubsystemError)
+	}
+
+	/// Inform the `Overseer` that that some block was finalized.
+	pub async fn block_finalized(&mut self) -> SubsystemResult<()> {
+		self.events_tx.send(Event::BlockFinalized).await.map_err(|_| SubsystemError)
+	}
+
+	/// Tell `Overseer` to shutdown.
+	pub async fn stop(&mut self) -> SubsystemResult<()> {
+		self.events_tx.send(Event::Stop).await.map_err(|_| SubsystemError)
+	}
+}
```

Comment:
We will also want a `send_external_message` thing for other services beyond our control (e.g. Babe block authorship task) to send messages to subsystems.

Reply:
The idea is to expand this method set, atm it contains `block_imported` and `block_finalized` that kind of demostrate how the outside world is supposed to send messages to subsystems